account_invoice_google_document_ai/views/account_move.xml

```diff
@@ -14,11 +14,19 @@
                     class="oe_highlight"
                     string="Send to OCR"
                     type="object"
-                    attrs="{'invisible': [('can_send_to_ocr', '=', False)]}"
+                    attrs="{'invisible': ['|', ('can_send_to_ocr', '=', False), ('ocr_sent', '=', True)]}"
+                />
+            <button
+                    name="ocr_process"
+                    class="btn-secondary"
+                    string="Send to OCR"
+                    type="object"
+                    attrs="{'invisible': ['|', ('can_send_to_ocr', '=', False), ('ocr_sent', '=', False)]}"
```

Comment:
@bizzappdev if the attachment will be changed, its impossible to rerun as normal user. cant we just c hange the button from Highlighted one to normal or add any other option to reset it ?